frontend/src/vendor/Segment.ts

```diff
@@ -37,7 +37,6 @@ export const initializeSegment = (): void => {
     "trackLink",
     "trackForm",
     "pageview",
-    "identify",
```

Comment:
This is (I think) just vendored third-party code, so we shouldn't modify this file unless it's necessary.

Reply:
Removed this change